cmd/update/cluster/command.go

```diff
@@ -0,0 +1,88 @@
+package cluster
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
+)
+
+const (
+	name = "cluster --name <cluster-name> --namespace <cluster-namespace> --release-version <release-version> --scheduled-time <scheduled-time> --provider <provider>"
+
+	shortDescription = "Schedule a cluster update."
+	longDescription  = `Schedule a cluster update.
+
+Updates given cluster with the provided values.
+
+Options:
+  --name <name>               		Name of the cluster to update.
+  --namespace <namespace>     		Namespace of the cluster.
+  --release-version <release-version>   Update the cluster to a release version. The release version must be higher than the current release version.
+  --scheduled-time <scheduled-time>     Scheduled time when cluster should be updated. The value has to be in RFC822 Format and UTC time zone.
+  --provider <provider> 			Name of the provider.`
```

Comment:
Unfortunately provider has to be provided, this is a design choice which was made in `pkg/data/domain/cluster`, I'm not happy with that but I don't want to rewrite the whole code base.

When we fetch a cluster via `get` function we automatically fetch the infra provider CR even you don't need it but that's how it is designed.